third-party/macosx-libevent-event-config.h

```diff
@@ -1,433 +1,513 @@
 /* event-config.h
- * Generated by autoconf; post-processed by libevent.
- * Do not edit this file.
+ *
+ * This file was generated by cmake when the makefiles were generated.
```

Comment:
Placed 10.10 SDK to `/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs` and used CMake, as it respects MACOSX_DEPLOYMENT_TARGET=10.10